cmd/root.go

```diff
@@ -51,6 +51,7 @@ var monitor bool
 var timeout time.Duration
 var log = logrus.New()
 var taskStatus = new(string)
+var instance = new(string)
```

Comment:
do we still need this?

Reply:
Yes we need it. This is the variable in which we capture, as string, the passed instanceNumbers. All the flags, used by other commands as well, are defined here. So I felt it is apt to define this flag here